scss-bundle.config.json

```diff
@@ -0,0 +1,10 @@
+{
+  "*1": "scss-bundle requires a config file in the Bazel execroot.",
+  "*2": "The config is used in the src/material:theming_bundle target",
```

Comment:
I don't like that `scss-bundle` now **requires** a config file since from a Bazel perspective.. it was more convenient to just pass flags & and sometimes needed for `$(location)`. Though there is no way around it.. we can create an issue to ask them to make the config file optional.. but it looks like they need it for more (i.e. determining node modules folder)